lib/promoted/ruby/client/request_builder.rb

```diff
@@ -29,7 +29,7 @@ def set_request_params args = {}
           @use_case                = Promoted::Ruby::Client::USE_CASES[request[:use_case]] || Promoted::Ruby::Client::USE_CASES['UNKNOWN_USE_CASE']
           @full_insertion          = args[:full_insertion]
           @user_info               = request[:user_info] || { :user_id => nil, :log_user_id => nil}
-          @timing                  = request[:timing] || { :client_log_timestamp => Time.now.to_i }
+          @timing                  = request[:timing] || { :client_log_timestamp => Time.now.to_i * 1000 }
```

Comment:
I'm okay if this is still floored or rounded to seconds.  We only currently use this when debugging.

Reply:
That's a good point, I changed it to (Time.now.to_f * 1000).to_i